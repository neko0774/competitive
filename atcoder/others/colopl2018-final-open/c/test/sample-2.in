11
1 3 6 10 15 18 15 10 6 3 1
