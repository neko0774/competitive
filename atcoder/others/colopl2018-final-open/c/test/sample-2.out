1
2
4
7
10
14
10
7
4
2
1
